tests/projects/test_databricks.py

```diff
@@ -146,7 +146,7 @@ def run_databricks_project(cluster_spec, **kwargs):
 
 def test_upload_project_to_dbfs(
     dbfs_root_mock, tmp_path, dbfs_path_exists_mock, upload_to_dbfs_mock
-):  # pylint: disable=unused-argument
+):
```

Comment:
`upload_to_dbfs_mock` is a pytest fixture. Removing it is a behavior change.

Reply:
There are obvious ones like `tmp_path`. If we find an unused `tmp_path` in tests, that's likely to be unused.